internal/pkg/cli/flag.go

```diff
@@ -224,7 +225,11 @@ Must be either "MySQL" or "PostgreSQL".`
 	taskGroupFlagDescription     = `Optional. The group name of the task. 
 Tasks with the same group name share the same set of resources. 
 (default directory name)`
-	taskImageTagFlagDescription = `Optional. The container image tag in addition to "latest".`
+	taskImageTagFlagDescription    = `Optional. The container image tag in addition to "latest".`
+	generateCommandFlagDescription = `Optional. Generate a command with a pre-filled value for each flag.
+To use a non-Copilot ECS service, specify --generate-cmd <cluster name>/<service name>.
```

Comment:
Q: we have fully surfaced the term "workload" to users at this point, right? If so, use "workload" here and in the next line instead of "service"?

Reply:
Good catch. I should update the flag description in this PR. I'd refrain from using the keyword `workload` because it doesn't appear anywhere formally in our doc site. I will just say "service or job".

The changes to `flag.go` in this PR is a result of #2201. I will rebase this branch to mainline once that one is merged. 
The real changes in this PR are located at `ecs/run_task_request.go` and `aws/stepfunctions`.